# Unreal path pointing to a staticmesh
string path_to_mesh
# Unreal path pointing to a material
string path_to_material
